.circleci/config.yml

```diff
@@ -76,6 +76,16 @@ jobs:
           keys:
             - v1-dependencies-{ { checksum "package.json" }}
 
+      - run:
+          name: Set build type
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "main" ]; then
+              echo "Running on main"
+              echo 'export BUILD_TYPE=release' >> $BASH_ENV
+            else
+              echo "Running on PR"
+              echo 'export BUILD_TYPE=debug' >> $BASH_ENV
+            fi
```

Comment:
I think this strategy could be simplified by adding the `--define build_type=` to the bazel rc for CI below. This way, the `build_type` would be applied to all bazel invocations automatically, instead of requiring you to manually define for each Bazel command.

Reply:
does that not apply to all CI builds regardless of main/PR

Reply:
nvm I get what you mean now